vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt

```diff
@@ -147,13 +181,23 @@ class VectorJitsiActivity : VectorBaseActivity<ActivityJitsiBinding>(), JitsiMee
         super.onDestroy()
     }
 
-//    override fun onUserLeaveHint() {
-//        super.onUserLeaveHint()
-//        jitsiMeetView?.enterPictureInPicture()
-//    }
+    override fun onUserLeaveHint() {
```

Comment:
Also, I think this method is only called when pressing home button, what about pressing back?

Reply:
Yes, if you press back, you leave the conference, and I think it is OK, no?

Reply:
I don't know, generally I want to leave only when explicitly pressing hang-up